### DISCUSSION TITLE
Kasa Camera Integration

### DISCUSSION THREAD

**Discussion by cincyfan1185** [plasticrake/homebridge-tplink-smarthome#267] [Q&A]
Any discussion to having Kasa Camera's included in the connection to Homekit? 

I have the bridge set-up through Hoobs, but that device doesn't seem to be pulling over.

**Comment by FatMacchiato** [comment]
I don't know if this is proper use of response field, but was wondering if you have found any solutions to integrate Kasa cams?  I have quite a few and would love to lump them into my HomeKit Home camera dashboard.
